app.js

```diff
@@ -258,7 +258,7 @@ function createViewer(ipcData) {
       autoHideMenuBar: true,
       show: false,
       titleBarStyle: 'hidden',
-      frame: process.platform !== 'win32',
+      frame: false,
```

Comment:
do we need to do the same thing on line 456?

Reply:
Done!